tests/ballerina-unit-test/build.gradle

```diff
@@ -171,21 +171,12 @@ test {
     systemProperty "java.util.logging.manager", "org.ballerinalang.logging.BLogManager"
 
     useTestNG() {
-        useDefaultListeners = true
         suites 'src/test/resources/testng.xml'
     }
 
     afterSuite { suite, result ->
         result.exception?.printStackTrace()
     }
-
-    beforeTest { descriptor ->
-        logger.lifecycle("Running test: " + descriptor)
-    }
-
-    testLogging {
-        events "passed", "skipped", "failed"
```

Comment:
Shouldn't we log the failed test? ?

Reply:
The failed tests will be logged without this config

Reply:
Ack.